pkgs/development/python-modules/cgal/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, python3
```

```suggestion
, python
```
because we want different packages for Python 3.9 and Python 3.10